contracts/upgradeable_contracts/RewardableValidators.sol

```diff
@@ -5,7 +5,7 @@ import "./BaseBridgeValidators.sol";
 
 contract RewardableValidators is BaseBridgeValidators {
 
-    event ValidatorAdded (address indexed validator, address reward);
+    event ValidatorAdded (address indexed validator);
```

Comment:
Do you think it make sense to move declarations of both events to the BaseBridgeValidators contract?

Reply:
Good catch! Yes, I think it makes sense. Done https://github.com/poanetwork/poa-bridge-contracts/pull/169/commits/9be3f38bdf2fa584a0b1367154abf33a6757ad99